Can't choose more then one doc target

In https://github.com/kerl/kerl/pull/336 support for building the new doc chunks were added by setting `DOC_TARGETS` when building the docs. This is great, and even better you can tweak said `DOC_TARGETS` using `KERL_DOC_TARGETS`.

I like having the man pages around, and also be able to use `h/1` in the shell would be nice. Unfortunately you can only build one such doc target, as the string concatenation here:

https://github.com/kerl/kerl/blob/cd6280cb7de71e0d3c3c61648aeca12e2b70cffb/kerl#L947-L948

Makes the build fail with ``make[3]: *** No rule to make target `chunks', needed by `docs'.  Stop.``, when `KERL_DOC_TARGETS="man chunks"`.

Manually running `make release_docs DOC_TARGETS="man chunks"` works like a charm though.

The solution is probably to avoid string concatenation altogether. I'm thinking either writing out the command in full every time it's run (maybe another small helper function?), or setting `DOC_TARGETS` as an exported environmental variable and hope `make` picks it up. The latter _should_ work on most systems, but if kerl wants to support stuff like AIX/Sun/Solaris/... then it might be a solution.

What do you think?

--- a/kerl
+++ b/kerl
@@ -971,15 +971,12 @@ _do_build() {
     if [ -n "$KERL_BUILD_DOCS" ]; then
         echo 'Building docs...'
         release=$(get_otp_version "$2")
-        cmd="make docs DOC_TARGETS=$KERL_DOC_TARGETS"
-        release_cmd="make release_docs DOC_TARGETS=$KERL_DOC_TARGETS RELEASE_ROOT=$KERL_BUILD_DIR/$2/release_$1"
-
-        if ! $cmd >>"$LOGFILE" 2>&1; then
+        if ! make docs "DOC_TARGETS=$KERL_DOC_TARGETS" >>"$LOGFILE" 2>&1; then
             show_logfile 'Building docs failed.' "$LOGFILE"
             list_remove builds "$1 $2"
             exit 1
         fi
-        if ! $release_cmd >>"$LOGFILE" 2>&1; then
+        if ! make release_docs "DOC_TARGETS=$KERL_DOC_TARGETS" "RELEASE_ROOT=$KERL_BUILD_DIR/$2/release_$1" >>"$LOGFILE" 2>&1; then
             show_logfile 'Release of docs failed.' "$LOGFILE"
             list_remove builds "$1 $2"
             exit 1
@@ -1302,8 +1299,7 @@ rehash
 ACTIVATE_CSH
 
     if [ -n "$KERL_BUILD_DOCS" ]; then
-        release_docs="make release_docs DOC_TARGETS=$KERL_DOC_TARGETS RELEASE_ROOT=$absdir"
-        if ! (ERL_TOP="$ERL_TOP" $release_docs >/dev/null 2>&1); then
+        if ! (ERL_TOP="$ERL_TOP" make release_docs "DOC_TARGETS=$KERL_DOC_TARGETS" "RELEASE_ROOT=$absdir" >/dev/null 2>&1); then
             echo "Couldn't install docs for Erlang/OTP $rel ($1) in $absdir"
             exit 1
         fi

